A28-xds-traffic-splitting-and-routing.md

```diff
@@ -175,10 +167,15 @@ message XdsRouting {
     // match_fraction specifies the percentage this route should be considered.
     // For fraction `N/D`, route is considered if `random(0,D) <= N`.
     //
+    // The number in this field specifies the numerator. Defaults to 0.  The
```

Comment:
It doesn't default to 0; it defaults to unset (or 1M, if you want to think of it that way).

Actually, I think you can just remove "Defaults to 0" here, since the default is described correctly in the next paragraph.

Reply:
Right. Removed. Thanks!